backend/apps/owasp/models/award.py

```diff
@@ -0,0 +1,246 @@
+"""OWASP app award model."""
+
+from __future__ import annotations
+
+from django.db import models
+
+from apps.common.models import TimestampedModel
+
+
+class Award(TimestampedModel):
+    """OWASP Award model.
+
+    Represents OWASP awards based on the canonical source at:
+    https://github.com/OWASP/owasp.github.io/blob/main/_data/awards.yml
+    """
+
+    class Meta:
+        db_table = "owasp_awards"
+        indexes = [
+            models.Index(fields=["category", "year"], name="owasp_award_category_year"),
+            models.Index(fields=["-year"], name="owasp_award_year_desc"),
+            models.Index(fields=["name"], name="owasp_award_name"),
+        ]
+        verbose_name = "Award"
+        verbose_name_plural = "Awards"
+        constraints = [
+            models.UniqueConstraint(
+                fields=["name", "year", "category"], name="unique_award_name_year_category"
+            )
+        ]
+
+    # Core fields based on YAML structure
+    category = models.CharField(
+        verbose_name="Category",
+        max_length=100,
+        help_text="Award category (e.g., 'WASPY', 'Lifetime Achievement')",
+    )
+    name = models.CharField(
+        verbose_name="Name",
+        max_length=200,
+        help_text="Award name/title (e.g., 'Event Person of the Year')",
+    )
+    description = models.TextField(
+        verbose_name="Description", blank=True, default="", help_text="Award description"
+    )
+    year = models.IntegerField(
+        verbose_name="Year",
+        null=True,
+        blank=True,
+        help_text="Year the award was given (null for category definitions)",
```

Comment:
The category part needs to be hardcoded. I don't think we should create a separate table for them as they rarely change.

Reply:
After removing that you can focus completely on award entities.